src/build/build_step.go

```diff
@@ -330,7 +330,12 @@ func moveOutputs(state *core.BuildState, target *core.BuildTarget) ([]string, bo
 	tmpDir := target.TmpDir()
 	outDir := target.OutDir()
 	for _, output := range target.Outputs() {
-		tmpOutput := path.Join(tmpDir, output)
+		var tmpOutput string
+		if output == target.Label.PackageName {
```

Comment:
this logic seems duplicated from the other new function. I think it should probably be a member function on `BuildTarget`